webcomponents-lite.js

```diff
@@ -53,9 +53,9 @@
     'URL/URL.js',
     'WeakMap/WeakMap.js',
     'MutationObserver/MutationObserver.js',
+    'Template/Template.js',
```

Comment:
That sounds right.

On Thu, Oct 29, 2015 at 4:42 PM, Christopher Joel <notifications@github.com>
wrote:

> In webcomponents-lite.js
> <https://github.com/webcomponents/webcomponentsjs/pull/429#discussion_r43458376>
> :
>
> > @@ -53,9 +53,9 @@
> >      'URL/URL.js',
> >      'WeakMap/WeakMap.js',
> >      'MutationObserver/MutationObserver.js',
> > +    'Template/Template.js',
>
> I chatted with Dan, and discovered that build.json pulls in ShadowDOM
> Polyfill, which has a separate HTMLTemplate.js implementation... so, I
> think build-lite.json is the only thing that needs to be modified?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/webcomponents/webcomponentsjs/pull/429/files#r43458376>
> .
>